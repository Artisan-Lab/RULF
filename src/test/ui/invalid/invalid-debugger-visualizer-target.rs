#![feature(debugger_visualizer)]

#[debugger_visualizer(natvis_file = "../foo.natvis")] //~ ERROR attribute should be applied to a module
fn main() {}
