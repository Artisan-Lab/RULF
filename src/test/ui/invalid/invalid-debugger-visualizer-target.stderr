error: attribute should be applied to a module
  --> $DIR/invalid-debugger-visualizer-target.rs:3:1
   |
LL | #[debugger_visualizer(natvis_file = "../foo.natvis")]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

